pkg/oomstore/types/apply/apply.go

```diff
@@ -0,0 +1,35 @@
+package apply
+
+import "io"
+
+type ApplyOpt struct {
+	R io.Reader
+}
+
+type Feature struct {
+	Kind        string `mapstructure:"kind"`
+	Name        string `mapstructure:"name"`
+	GroupName   string `mapstructure:"group-name"`
+	DBValueType string `mapstructure:"db-type-value"`
```

Comment:
`db-value-type` or `db_value_type`?

Reply:
hmm, I am not sure which is better... The - is currently used uniformly